src/modules/layoutUiRules/rules.js

```diff
@@ -39,6 +39,22 @@ const defaultRules = {
     },
 }
 
+const dualCategoryRules = {
```

Comment:
Let's stick to the "Two category" term, so `twoCategoryRules`

Reply:
Renamed all functions related to two category.